modular_skyrat/master_files/code/modules/pain_shock/stat_handler.dm

```diff
@@ -0,0 +1,58 @@
+/mob/living/carbon/set_stat(new_stat)
+	. = ..()
+	if(should_be_critical(new_stat))
+		ADD_TRAIT(src, TRAIT_HANDS_BLOCKED, STAT_TRAIT)
+		ADD_TRAIT(src, TRAIT_CRITICAL_CONDITION, STAT_TRAIT)
+		if(flow_rate <= 80)
+			ADD_TRAIT(src, TRAIT_IMMOBILIZED, TRAIT_KNOCKEDOUT)
+			ADD_TRAIT(src, TRAIT_FLOORED, STAT_TRAIT)
+		if(stat == DEAD)
+			consider_the_score(new_stat)
+	else
+		switch(new_stat)
+
+			if(CONSCIOUS && !should_be_critical(new_stat))
```

Comment:
logic statements in a switch is bad, the code wont compile

Reply:
the code does compile like this and it works

Reply:
The code literally interprets this as TRUE, and will only be run when  new_stat is TRUE (aka 1, byond doesn't differentiate bools from integers).